chimneyCats/src/main/scala/io/scalaland/chimney/cats/package.scala

```diff
@@ -0,0 +1,35 @@
+package io.scalaland.chimney
+
+import _root_.cats.data.Validated
+import _root_.cats.kernel.Semigroup
+
+import scala.collection.compat._
+
+package object cats {
+
+  implicit def TransformerFValidatedSupport[EE: Semigroup]: TransformerFSupport[Validated[EE, +*]] =
+    new TransformerFSupport[Validated[EE, +*]] {
```

Comment:
Maybe add cats.data.Ior for semigroup A instance too?

Reply:
Looks like a good idea 👍 

But I'm struggling with reasonable semantics of
```scala
def product[A, B](fa: Ior[EE, A], fb: => Ior[EE, B]): Ior[EE, (A, B)]
```
for few cases:
- when `fa = Left(e)` and `fb = Right(b)` - we cannot have `Ior.Both`, as we can't construct right pair (`(A, B)`), since we only have `B`; `Ior.Left` is the only result we can have here
- when `fa = Left(e1)` and `fb = Both(e2, b)` - we can combine `e1` and `e2` using Semigroup instance, but also we cannot have `Ior.Both` as a result
It also includes symmetrical cases.

I'm not really familiar with using `Ior`, but I'm afraid that `product` can't be implemented with substantial information loss. The question is if such semantics make sense at all?